src/vis/vis.js

```diff
@@ -399,11 +399,6 @@ var VisModel = Backbone.Model.extend({
     this.trigger('invalidateSize');
   },
 
-  centerMapToOrigin: function () {
```

Comment:
This method is being used by deep-insights.js ([here](https://github.com/CartoDB/deep-insights.js/blob/7f106bf59c969999fd635789b100b2a311a26d2b/src/api/create-dashboard.js#L111)).